packages/design-system/src/components/N8nCallout/Callout.vue

```diff
@@ -0,0 +1,163 @@
+<template>
+	<div :class="classes" role="alert">
+
+		<div :class="$style['message-section']">
+			<div :class="$style.icon">
+				<n8n-icon
+					v-bind="$attrs"
+					:icon="getIcon"
+					:size="theme === 'secondary' ? 'medium' : 'large'"
+				/>
+			</div>
+			<n8n-text
+				size="small"
+				v-bind="$attrs"
+				:bold="theme === 'secondary'"
+			>
+				{{ message }}
+			</n8n-text>
+			<span
+				v-if="actionText"
+				:class="$style['action-text']"
+				@click="$emit('action-text-click')"
+			>
+				{{ actionText }}
+			</span>
+		</div>
+
+		<n8n-link
+			v-if="trailingLinkText && trailingLinkUrl"
+			:to="trailingLinkUrl"
+			size="small"
+			theme="secondary"
+			:bold="true"
+			:underline="true"
+		>
+			{{ trailingLinkText }}
+		</n8n-link>
```

Comment:
To make this component even more reusable for the future, we should have a slot that we can use to place content at the top-right.

~~~
<slot name="actions" />
~~~

Then, we'd be able to use this callout component and put anything (not just a link) at the top right in the future.

~~~html
<n8n-callout>
  <template #actions>
                 <n8n-link
			:to="/something"
			size="small"
			theme="secondary"
			:bold="true"
			:underline="true"
		>
			Some text
		</n8n-link>
  </template>
</n8n-callout>
~~~